### ISSUE TITLE
Look into reviews_by_reviewer method (is it still needed?)

### ISSUE THREAD

**Issue by lburgess07** [Micah-Gafford/expertiza#10]
It seems unlikely that reviews_by_reviewer is still needed.  It refers to reviews of this assignment_participant.  But for the last 5 years or so, all reviews have been of assignment_teams, not assignment_participants.
Ditto for get_logged_in_reviewer_id and files.
(In general, method names beginning with get_ are not Ruby-like, and should be changed.)

**Comment by lburgess07** [comment]
reviews_by_reviewer appears in 4 different files, assignment_participant, collusion_cycle.rb (both model classes), and then in the tests for each model... but collusion_cycle.rb has a comment at the top that says 
- new class created to handle code related to cycles between reviews
- This code was previously in AssignmentParticipant and is not being used anywhere yet

Moreover, I searched for any instances of CollusionCycle in the application and there are no references anywhere in the app - so all of this code may be safe to delete, if the functionality isn't going to be used in the future. 

So collusioncycle seems to be depndent on this reviews_by_reviewer, but CollusionCycle is not used anywhere else in the application. what should we do?

**Comment by lburgess07** [comment]
so basically, i've determined that this is probably not needed, but need to inquire about possible future usage of code

**Comment by lburgess07** [comment]
remove if not used!
Dr. Gehringer will follow up with Young about ColusionCycle and if it gets used.

**Comment by lburgess07** [comment]
Dr. Gehringer has instructed me to remove the reviews_by_reviewer method in assignment_participant. This is because it is only called in collusion_cycle.rb, which the class itself is not called or used anywhere in the application and hasn't been used in a while. 

I have made the following changes on _reviews_by_reviewer_ branch:

- Removed method definition from assignment_participant.rb
- Removed the rspec test for this method from spec/models/assignment_participant_spec.rb

Note: As instructed, I did not change or delete any of the calls in collusion_cycle.rb or collusion_cycle_spec.rb which both call this method several times.
